kuma/javascript/src/payments/index.jsx

```diff
@@ -0,0 +1,363 @@
+                                            rel="noopener noreferrer"
+                                            href={urls.email}
+                                        >
+                                            {gettext('contact support')}
+                                        </a>
+                                    }
+                                />
+                            </p>
+                        </ListItem>
+
+                        <ListItem
+                            title="What are the payment terms?"
+                            number="13"
+                        >
+                            <p>
+                                <Interpolated
+                                    id={gettext(
+                                        'Please read our <paymentLink /> for more information.'
+                                    )}
+                                    paymentLink={
+                                        <a href={urls.terms}>
+                                            {gettext('payment terms')}
+                                        </a>
+                                    }
+                                />
+                            </p>
+                        </ListItem>
+                        <ListItem
+                            title="Does deleting my MDN account cancel my monthly subscription?"
+                            number="14"
+                        >
+                            <p>
+                                {gettext(
+                                    'When you request to delete your account we will also cancel your monthly subscription and not charge you for subsequent months.'
+                                )}
+                            </p>
+                        </ListItem>
+                    </ol>
+                </section>
+            </main>
+            <Footer />
+        </>
+    );
+}
+
+// In order to use new URL() with relative URLs, we need an absolute base
+// URL. If we're running in the browser we can use our current page URL.
+// But if we're doing SSR, we just have to make something up.
+const BASEURL =
+    typeof window !== 'undefined' && window.location
+        ? window.location.origin
+        : 'http://ssr.hack';
+
+export class PaymentsRoute extends Route<PaymentsRouteParams, null> {
+    locale: string;
+
+    constructor(locale: string) {
+        super();
+        this.locale = locale;
+    }
+
+    getComponent() {
+        return PaymentsLandingPage;
+    }
+
+    match(url: string): ?PaymentsRouteParams {
+        const path = new URL(url, BASEURL).pathname;
+        const paymentsPath = `/${this.locale}/payments`;
+        const regex = new RegExp(paymentsPath, 'g');
+
+        if (regex.test(path)) {
+            return {
+                locale: this.locale
+            };
+        }
+        return null;
+    }
+
+    fetch() {
+        return Promise.resolve(null);
```

Comment:
I think this should never run because we never need to do client-side fetch. So you can replace this with 
```suggestion
        throw new Error('Payments should never need to post-fetch more data');
```